scss/note-list.scss

```diff
@@ -86,6 +86,7 @@
 }
 
 .note-list-item {
+	position: relative;
```

Comment:
We could probably do some fancy flexbox here, but this works fine. No reason to make it complicated.